web/src/app/admin/AdminSection.tsx

```diff
@@ -86,8 +86,19 @@ export default function AdminSection(props: AdminSectionProps): JSX.Element {
             >
               <ListItemText
                 className={classes.listItemText}
-                primary={f.label}
-                secondary={f.description}
+                primary={(f.deprecated ? '* ' : '') + f.label}
+                secondary={
+                  f.deprecated ? (
+                    <React.Fragment>
+                      <b>Deprecated:</b> {f.deprecated}
+                      <br />
+                      <br />
+                      {f.description}
+                    </React.Fragment>
```

Comment:
I haven't verified how this looks, but
```suggestion
                    <React.Fragment>
                      <Alert severity="warning" sx={{ pb: 1 }}>
                        <AlertTitle>Deprecated</AlertTitle>
                        {f.deprecated}
                      </Alert>
                      {f.description}
                    </React.Fragment>
```

Reply:
Nice! Yeah I like that a lot, the only thing is maybe adding padding around it, and if we wanted to keep it skinnier, we could omit using the `AlertTitle` and just do `<Alert severity='warning'>Deprecated: {f.deprecated}</Alert>`

Reply:
![image](https://user-images.githubusercontent.com/595010/177657920-043bcdb7-27ee-46a8-bf08-81d305dcb66f.png)

Reply:
I think a good follow-up would be to have the deprecated ones hidden by default with maybe a checkbox somewhere "Show deprecated fields". For now, some of these fields are still used if the flag isn't set, so we can keep that out of scope.